src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.7.json

```diff
@@ -0,0 +1,199 @@
+    "DisplayType": "Pages",
+    "RequestedFeedbackAnswered": null,
+    "Pages": [
+        {
+            "PageId": "7700",
+            "SequenceId": null,
+            "SectionId": null,
+            "Title": "Upload your organisation's contract for services template with employers",
+            "LinkTitle": "Upload your organisation's contract for services template with employers",
+            "InfoText": "",
+            "Questions": [
+                {
+                    "QuestionId": "DAT-770",
+                    "QuestionTag": null,
+                    "Label": "Upload your organisation's policy for professional development of employees",
+                    "ShortLabel": "",
+                    "InputClasses": null,
+                    "QuestionBodyText": "<p class='govuk-body'>This must include how your organisation will continuously:</p><ul class='govuk-list govuk-list--bullet'><li>improve its employees sector expertise, skills and performance</li><li>improve its employees teaching and training knowledge, skills and performance</li><li>Improve its employees skills and performance</li><li>ensure learning objectives are tailored to meet employee, customer and organisation needs</li><li>monitor employees professional development</li></ul><p class='govuk-body'>It must also include your organisation's vision for professional development of employees.</p><p class='govuk-body'>The file must be a PDF and smaller than 5MB.</p>",
+                    "Hint": "Upload a file",
+                    "Input": {
+                        "Type": "FileUpload",
+                        "InputClasses": null,
+                        "Options": [],
+                        "Validations": [
+                            {
+                                "Name": "Required",
+                                "Value": "",
+                                "ErrorMessage": "Select your organisation's policy for professional development of employees"
+                            },
+                            {
+                                "Name": "FileType",
+                                "Value": "pdf",
+                                "ErrorMessage": "The selected file must be a PDF"
+                            }
+                        ],
+                        "DataEndpoint": null
+                    },
+                    "Order": null,
+                    "Value": "",
+                    "ErrorMessages": null
+                }
+            ],
+            "PageOfAnswers": [],
+            "Next": [
+                {
+                    "Action": "NextPage",
+                    "ReturnId": "7710",
+                    "Conditions": [],
+                    "ConditionMet": false
+                }
+            ],
+            "Complete": false,
+            "AllowMultipleAnswers": false,
+            "Order": null,
+            "Active": true,
+            "NotRequiredConditions": [
```

Comment:
Can this empty array be removed? If it's generated by the QnA tool, not a problem

Reply:
@David-B-Read Yes, it’s generated by QnA Config Tool. I also checked some other section’s json files and they all have empty arrays, so to be consistent I will leave it as that.